packages/ocean-react/src/Alert/stories/Alert.stories.mdx

```diff
@@ -0,0 +1,119 @@
+
+### Simple alerts
+
+The alert offers four diferents default types that set a icon and color.
+
+<Canvas>
+  <Story name="default">
+    <Alert>This is an informational message.</Alert>
+  </Story>
+  <Story name="warning">
+    <Alert type="warning">This is an informational message.</Alert>
+  </Story>
+  <Story name="error">
+    <Alert type="error">This is an informational message.</Alert>
+  </Story>
+  <Story name="success">
+    <Alert type="success">This is an informational message.</Alert>
+  </Story>
+</Canvas>
+
+### Description
+
+You can use the `title` property to display a formatted title above the alert message.
+
+<Canvas>
+  <Story name="default-with-title">
+    <Alert title="Default Title">
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+    </Alert>
+  </Story>
+  <Story name="warning-with-title">
+    <Alert type="warning" title="Warning Title">
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+    </Alert>
+  </Story>
+  <Story name="error-with-title">
+    <Alert type="error" title="Error Title">
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+    </Alert>
+  </Story>
+  <Story name="success-with-title">
+    <Alert type="success" title="Success Title!">
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+    </Alert>
+  </Story>
+</Canvas>
+
+### Custom Icon
+
+You can also use the `icon` property to custom svg icon.
+
+<Canvas>
+  <Story name="default-with-custom-icon">
+    <Alert
+      title="Alert with a custom icon!"
+      icon={<StarOutline size={24} stroke="#5872f5" />}
+    >
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+    </Alert>
+  </Story>
+</Canvas>
+
+## CSS
+
+| Global class           | Description                                               |
+| ---------------------- | --------------------------------------------------------- |
+| .ods-alert             | Styles applied to the root element.                       |
+| .ods-alert--has-header | Styles applied to the root element with a title proprity. |
+| .ods-alert--error      | Styles applied to the root element with a error type.     |
+| .ods-alert--warning    | Styles applied to the root element with a warning type.   |
+| .ods-alert--success    | Styles applied to the root element with a success type.   |
+| .ods-alert\_\_header   | Styles applied to the header element.                     |
+| .ods-alert\_\_icon     | Styles applied to the icon element.                       |
+| .ods-alert\_\_title    | Styles applied to the title element.                      |
+
+If that's not sufficient, you can check the [implementation of the component](https://github.com/ocean-ds/ocean-web/blob/master/packages/ocean-react/src/Alert/Alert.tsx) for more detail.
+
+## Playground
+
+<Canvas>
+  <Story name="Playground">
+    {(props) => (
+      <Alert {...props}>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+      </Alert>
+    )}
+  </Story>
```

Comment:
```suggestion
  <Story
    name="Playground"
    args={{
      title: 'My title',
      type: 'default',
      children: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit.',
    }}
    argTypes={{
      icon: { control: null },
    }}
  >
    {(props) => <Alert {...props} />}
  </Story>
```

Reply:
just to don't show the icon input?

Reply:
icon is an element, and for avoid, users complain when they inform this field I think we can't disable for the playground.
See what i mean:

https://user-images.githubusercontent.com/3240432/129579789-aac16fc2-c819-4afa-9495-492a76553b15.mp4

Reply:
Similar to what we've done in the `component` field of the button.
![image](https://user-images.githubusercontent.com/3240432/129580077-7ee6b7d7-af26-4ee3-83c4-11c0c9049d83.png)